commands2/pidcommand.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) FIRST and other WPILib contributors.
+# Open Source Software; you can modify and/or share it under the terms of
+# the WPILib BSD license file in the root directory of this project.
+from __future__ import annotations
+
+from typing import Any, Callable
+
+from .command import Command
+from .subsystem import Subsystem
+
+from wpimath.controller import PIDController
+
+
+class PIDCommand(Command):
+    """
+    A command that controls an output with a PIDController. Runs forever by default - to add
+    exit conditions and/or other behavior, subclass this class. The controller calculation and output
+    are performed synchronously in the command's execute() method.
+
+    This class is provided by the NewCommands VendorDep
+    """
+
+    def __init__(
+        self,
+        controller: PIDController,
+        measurementSource: Callable[[], float],
+        setpoint: float,
+        useOutput: Callable[[float], Any],
+        *requirements: Subsystem,
+    ):
+        """
+        Creates a new PIDCommand, which controls the given output with a PIDController.
+
+        :param controller: the controller that controls the output.
+        :param measurementSource: the measurement of the process variable
+        :param setpoint: the controller's setpoint
+        :param useOutput: the controller's output
+        :param requirements: the subsystems required by this command
+        """
+        super().__init__()
+        if controller is None:
```

Comment:
It's a bit weird that you're checking all of these arguments to see if they're None. I know this is required in Java, but in python I think we should either do no check or check to see if the type is correct.